c++/naive_pattern_searching.cpp

```diff
@@ -0,0 +1,43 @@
+#include <bits/stdc++.h>
```

Comment:
You can change this to use vector, iostream and string just to be more precise on what libraries we use.